### PR TITLE
Another round of fixes for the CI system

### PR DESCRIPTION
**Pull request by rjoleary**
Signed-off-by: Ryan O'Leary <ryanoleary@google.com>

### CODE REVIEW

File: seeed/beaglev/flash.config [hunk 1]
```diff
@@ -0,0 +1,2080 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Linux/x86 5.7.0 Kernel Configuration
```
Comment (rjoleary) [inline seeed/beaglev/flash.config:3]: This isn't right because it's x86, but where should the flash.config come from?

Comment (rminnich) [reply seeed/beaglev/flash.config:3]: it's not right but ... this config is tested and works. Weird that it said that.

**Review by rminnich — COMMENTED** [review]
This is pretty fantastic :-)

File: seeed/beaglev/flash.config (hunk 1 above)
Comment (rjoleary) [reply seeed/beaglev/flash.config:3]: I just copied this config from apu2, it is definitely wrong

**Review by rminnich — DISMISSED** [review]

**Review by rminnich — APPROVED** [review]

**Pull request merged**